pages/1.10/deploying-services/private-docker-registry/index.md

```diff
@@ -230,3 +230,43 @@ Follow these steps to add your Docker registry credentials to the [DC/OS Enterpr
       ```
 
    The Docker image will now pull using the provided security credentials.
+
+<a name="tarball-instructions"></a>
+# Pushing a custom image to the private repository from tarball
+
+In some cases, you might have obtained a docker image in a form of a `.tar` archive (ex. if you asked your sales representative for an enterprise version of marathon). To deploy that image on your registry, follow these steps:
```

Comment:
`you obtained a Docker image`, `marathon -> Marathon`, `to your registry`

Reply:
✅